### DISCUSSION TITLE
Make check marks more visible ✅

### DISCUSSION THREAD

**Discussion by rwp0** [community/community#139349] [Mobile]
As employed in Pull Request templates.

Grey on grey is less prominent in Dark Mode.

So I've to look really close and with extra effort to see which one is selected in the example below:

![Screenshot_20240922_083018_GitHub.jpg](https://github.com/user-attachments/assets/f9613610-05e9-47a7-a143-36e4a23d3620)

Perhaps make it Green on Gray?

Thanks

**Comment by kangarruu** [comment]
@rwp0 Thanks for writing in! Is this on iOS or Android? I can repro this behavior on Android but it seems to be a transient stage in the markdown list behavior that then resolves into white on blue after a second or so. Does the grey on grey persist for you?

**Reply by rwp0** [reply]
Hello @kangarruu,
It's Android, sorry forgot to specify (Android 14 / One UI 6.1)
And yes it persists for me on dark background

**Comment by kangarruu** [comment]
Thanks for those details @rwp0! Would you be able to post a video of this happening? Is this occurring in light mode as well or specifically in dark mode?

**Reply by rwp0** [reply]
I'll post a video, never use the light mode (will attempt at checking if switching is supported natively by GitHub app instead of globally on Android).

**Reply by rwp0** [reply]
Oh, I remember checking yesterday, using Screen Recorder is not allowed while on the GitHub app.

> Can't record screen due to security policy.

Or it might be due to something else on my phone.
